crates/re_viewer/src/ui/view_spatial/scene/scene_part/images.rs

```diff
@@ -277,24 +278,53 @@ impl ImagesPart {
             return Err(format!("Couldn't fetch pinhole extrinsics at {pinhole_ent_path:?}"));
         };
 
-        // TODO(cmc): automagically convert as needed for non-natively supported datatypes?
-        let data = match &tensor.data {
-            // NOTE: Shallow clone if feature `arrow` is enabled, full alloc + memcpy otherwise.
-            TensorData::U16(data) => DepthCloudDepthData::U16(data.clone()),
-            TensorData::F32(data) => DepthCloudDepthData::F32(data.clone()),
-            _ => {
-                return Err(format!(
-                    "Tensor datatype {} is not supported for backprojection",
-                    tensor.dtype()
-                ));
-            }
+        let (height, width) = (tensor.shape()[0].size, tensor.shape()[1].size);
```

Comment:
You need to check that this is a 2D tensor first, or use `tensor.image_height_width_channels`